# MicrobeTaxaSARS2

### What is the problem?


### Why should we solve it?


### What is MicrobeTaxaSARS2?


### Workflow for MicrobeTaxaSARS2
Here is the workflow for the pipeline:
![Pipleline Workflow for Metagenomic Analysis of CoVID patients' microbiome](img/workflow.png)

### How to use PubRunner
Installation options:
We will provide a docker image for installing and using SARS2MicrobeTaxa.

The Docker image will contains SARS2MicrobeTaxa and it will be downloadable from the Docker Hub.

docker pull <!-- omicscodeathon/microbetaxasars2 -->: Command to pull the image from the DockerHub
docker run <!-- omicscodeathon/microbetaxasars2 -->: Run the docker image from the command line


### Methods

In this project, we analysed Whole Genome Sequences (WGS) from 5 CoVID-19 patients and the dataset used was obtained from the Sequence Read Achived. These were fecal samples from CoVID-19 patients. We used 5 control datasets to compare with the cases. For the 16S data analysis, we also analysed 16S rRNA sequences from saliva samples.

### Results
Results of the analysis are in the [output folder](output/).

### Team
Sophia Sei (sofiasehli00@gmail.com)
Casey Eddington (eddin022@umn.edu)
Zainab El Ouafi (zainab.elouafi@gmail.com)
Soumaya JBARA (soumaya.jbara@um5r.ac.ma)
Kasambula Arthur Shem (arthurshem86@gmail.com)
Ayorinde Afolayan (afolayanayorinde@gmail.com)
Islam El Jaddaoui (is.eljaddaoui@gmail.com)
Olaitan Awe (laitanawe@gmail.com)

### References
